### PR TITLE
deploy(contracts): uses @aragon/osx-commons-config for network information

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

replaces `packages/contracts/networks.json` with a `packages/contracts/networks.ts` that extends the networks exported by  `@aragon/osx-commons-configs` 

Task ID: [OS-925](https://aragonassociation.atlassian.net/browse/OS-925)

## Type of change

See the framework lifecylce in `packages/contracts/docs/framework-lifecycle` to decide what kind of change this pull request is.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [ ] I have updated the `UPDATE_CHECKLIST` file in the root folder.

[OS-925]: https://aragonassociation.atlassian.net/browse/OS-925?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,55 +14,73 @@
 
 # Aragon OSx Protocol
 
-The Aragon OSx protocol is the foundation layer of the new Aragon stack. It allows users to create, manage, and customize DAOs in a way that is lean, adaptable, and secure.
+The Aragon OSx protocol is the foundation layer of the new Aragon stack. It
```
Comment (Michael-A-Heuer) [inline README.md:17]: What has caused this change in the context of this PR? If prettier, why not earlier?
IMO, the `.md` got less readable now

**Review by Michael-A-Heuer — DISMISSED** [review]
LGTM. `README.md` reformatting might be unwanted.

File: README.md (hunk 1 above)
Comment (Rekard0) [reply README.md:17]: Strange, But i review this file in the branch it looks fine,
why here this is happening here?

File: packages/contracts/networks.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import {NetworkConfigs, networks} from '@aragon/osx-commons-configs';
+
+export type ContractsNetworkConfig = {
+  deploy: string[];
+};
+
+const networkExtensions: NetworkConfigs<ContractsNetworkConfig> = {
+  mainnet: {
+    ...networks.mainnet,
+    deploy: ['./deploy/update/to_v1.3.0', './deploy/verification'],
```
[the remark this answers is not in the document]
Comment (Rekard0) [reply packages/contracts/networks.ts:10]: Shouldn't we all add a point to the checklist ?

File: packages/contracts/utils/etherscan.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
+import networks from '../networks';
 import fs from 'fs';
 import HRE from 'hardhat';
-import path from 'path';
 import {file} from 'tmp-promise';
```
Comment (Rekard0) [inline packages/contracts/utils/etherscan.ts:5]: why the order of the import is not from far to close?

**Review by Rekard0 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (mathewmeconry) [reply README.md]: this is my default formatter that I use. Reverted this change

**Pull request merged**